smartapps/fireboy1919/milight-light.src/milight-light.groovy

```diff
@@ -35,8 +35,9 @@ def nameMiLights() {
 	dynamicPage(name: "nameMiLights", title: "MiLight Wifi Hub Setup", uninstall: true, install: true) {
         section("Light") {
             input "miLightName", "text", title: "Light name", description: "i.e. Living Room", required: true, submitOnChange: false
-            input "code", "number", title: "Code", required: true, description: "Optional: will be autoassigned"
+            input "code", "number", title: "Code", required: false, description: "Optional: will be autoassigned"
```

Comment:
Since this isn't actually autoassigned (because I couldn't figure out a way to do that after an hour or so of looking), this is actually needed, and the app was throwing exceptions when I didn't require it.  So this would produce a bug.

Reply:
I guess i felt like if something were optional, it shouldn't be required... :)